### PR TITLE
added electricity EFs from UK suppliers

### PR DESCRIPTION
**Pull request by gpantelidou**
**Is this a new addition or an update to an existing factor?**

- [x] New
- [ ] Update

**What category or categories does it affect?**

Electricity

**What?**

new fuel mix EFs from electricity suppliers in UK for the latest year (most 2022, a few 2021) + new sub-id for the activity_id structure. 

**Why?**

good dataset for UK customers

### CODE REVIEW

**Review by maryarbab — DISMISSED** [review]
Looks clean!

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -20982,36 +20982,36 @@ f431a26a-5938-40f1-9776-878e996a5c8f,Energy,Electricity,electricity-energy_sourc
...
+65eaec9f-a8c2-49ef-bb44-eaf77be191d2,Agriculture/Hunting/Forestry/Fishing,Arable Farming,arable_farming-fertilizer_ammonium_based_combined_climate_direct,Direct emissions from ammonium based fertilizer application in combined climate,kg,not-supplied,not-supplied,0.0,0.0,0.0157,not-supplied,not-supplied,,,use_phase,IPCC,2019 Refinement to the 2006 IPCC Guidelines for National Greenhouse Gas Inventories,2019,2019,,GLOBAL,,Climatiq,2023/02/10,"Direct emissions from N applications. From IPCC ""2019 Refinement to the 2006 IPCC Guidelines for National Greenhouse Gas Inventories"". Chapter 11 Equation 11.2. Wet / dry climate country match from: Menegat S. Ledo A. & Tirado R. Greenhouse gas emissions from global production and use of nitrogen synthetic fertilisers in agriculture. Sci Rep 12 14490 (2022).",https://www.ipcc.ch/report/2019-refinement-to-the-2006-ipcc-guidelines-for-national-greenhouse-gas-inventories/
```
Comment (TimCroker) [inline OpenEmissionFactorsDB.csv:20985]: What are the extra " for e.g.
from IPCC ""2019 Refinement to the 2006 IPCC Guidelines for National Greenhouse Gas Inventories"".

Comment (TimCroker) [reply OpenEmissionFactorsDB.csv:20985]: See also other lines on arable farming

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -21754,3 +21754,33 @@ dc0ba7ab-9069-4293-8818-7ed78e3675a0,Energy,Fuel,fuel_type_natural_gas-fuel_use_
...
+,Energy,Electricity,electricity-energy_source_grid-supplier_boost,Electricity supplied from grid - supplier Boost,kWh,not-supplied,0.261,0.261,not-supplied,not-supplied,not-supplied,not-supplied,,2,electricity_generation,Electricity Info,Fuel Mix of UK Domestic Electricity Suppliers,2022,2022,,GB,p,Climatiq,2023/03/01,Emission intensity of fuel mix by a specific electricity supplier in GB. Published by the ElectricityInfo.org and retrieved from the table Fuel Mix of UK Domestic Electricity Suppliers for the year 2022. NOTE: this value is provided in CO2 only and does not include the impact of other GHGs.,https://electricityinfo.org/fuel-mix-of-uk-domestic-electricity-suppliers/
```
[the remark this answers is not in the document]
Comment (TimCroker) [reply OpenEmissionFactorsDB.csv:21757]: (applies to all entries)

Comment (TimCroker) [inline OpenEmissionFactorsDB.csv:21757]: Change year-valid to the year before publishing for all i.e. 2022 publish = 2021 valid

**Review by TimCroker — CHANGES_REQUESTED** [review]

**Review by TimCroker — APPROVED** [review]

**Review by maryarbab — APPROVED** [review]

**Pull request merged**